steps/container_steps.py

```diff
@@ -118,7 +118,7 @@ def start_container_with_args_and_env(context, pname="java"):
 @when(u'container is started as uid {uid}')
 @when(u'container is started as uid {uid} with process {pname}')
 def start_container(context, uid, pname="java"):
-    if uid < 0:
+    if uid < "0":
```

Comment:
This looks weird to me, can you explain?

Reply:
its because in python3 you cannot compare string with int. but if you compare numbers in like "-1", "1" with "0" you get correct comparison, so I take this approach instead of converting args to ints and dealing with exceptions and so.